Appraisals

```diff
@@ -9,29 +9,36 @@ def ruby_version?(version)
     Gem::Version.new(RUBY_VERSION) < Gem::Version.new(full_version).bump
 end
 
+def add_appraise(group, &block)
+  # Specify the environment variable APPRAISAL_GROUP to load only a specific appraisal group.
+  if ENV['APPRAISAL_GROUP'].nil? || ENV['APPRAISAL_GROUP'] == group
+    appraise(group, &block)
+  end
+end
```

Comment:
I suggest overriding the built-in `appraise` method here, in order to keep this file looking like a plain appraisal file going forward.

I suggest adding `alias appraise add_appraise` after you declare this method, and reverting the method name changes that follow up in this file.

Reply:
Ok, I think you mean to alias the original method and then define a new method called appraise. Since this file is evaled by appraisal I'm unsure if `super` would work with just a override so I'm trying the alias.